File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/ContactHeldItemTransferChanceModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class ContactHeldItemTransferChanceModifierItem extends PokemonHeldItemModifierItem {

    public static final String TARGET = "ContactHeldItemTransferChanceModifierType";

    private int chancePercent;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/ContactHeldItemTransferChanceModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

class ContactHeldItemTransferChanceModifierItem : PokemonHeldItemModifierItem() {

    companion object {
        const val TARGET = "ContactHeldItemTransferChanceModifierType"
    }

    var chancePercent: Int = 0
}